src/app/public/modules/media-query/media-query.service.ts

```diff
@@ -70,6 +89,10 @@ export class SkyMediaQueryService implements OnDestroy {
     this.currentSubject.complete();
   }
 
+  /**
+   * Suscribes to screen size changes.
```

Comment:
Hmmm... Are you sure you don't have it? It should look like this: 
![Screen Shot 2020-08-31 at 12 11 58 PM](https://user-images.githubusercontent.com/18354182/91741774-30c9dc00-eb83-11ea-942b-57defdcdaddf.png)

I removed the class information, because it's now dynamically linked. The user should be able to click it and get all the pertinent information.

Reply:
I see that entry now. Not sure how I missed it before. Sorry about that. Good call on removing the long explanation in preference for the linked info. Thanks!